extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -384,6 +384,7 @@ export const ManageAssetRow = ({
         assetIcons={code !== "XLM" ? { [canonicalAsset]: image } : {}}
         code={code}
         issuerKey={issuer}
+        isSorobanToken={!!(issuer && isSorobanIssuer(issuer))}
```

Comment:
hm, looks like in these instances, you're always defining `isSorobanToken` based on the `issuerKey` value.

Would it be cleaner to have this check inside `AssetIcon`? Maybe something like:

`isSorobanToken = !isSorobanToken ? isSorobanIssuer(issuerKey) : isSorobanToken`

I'm not sure if this is actually better, but it allows us to do the ` isSorobanIssuer(issuerKey)` logic in one place while also allowing a dev to override. I see that the only other place we're actually passing `isSorobanToken` is when we deal with balances in `AccountAssets` 🤔

Reply:
yeah 100%, not even sure why I went the prop route so I'll refactor.

Reply:
refactored in a6f66dba1025348d5e6c8fadf4006e0cd8a86376